src/web/src/pages/Main.vue

```diff
@@ -156,6 +183,21 @@ export default {
       // This must be vm.set since we're adding a property onto an object
       this.$set(this.selectedCourses, course.id, course);
       this.scheduler.addCourse(course);
+
+      if(this.userID){
+        const info = {'cid':course.name, 'semester':this.currentSemester, 'uid':this.userID};
+
+        console.log(info);
+        addStudentCourse(info)
```

Comment:
I'm not sure if security is required for this PR, but it would be very easy to `spoof` my UID and add courses to other users due to a lack of checking. After #111 is merged in, it won't be too hard to secure this route.

Reply:
No one gave me a way of getting the uid, no matter how many times I asked, so I just put it in cookies. Feel free to fix this, but I can't do better with the information I have been given...

Reply:
That is fine, no problem.